Cargo.toml

```diff
@@ -33,6 +33,8 @@ tracing-subscriber = "0.3"
 protobuf-src = "1.1.0"
 prost-build = "0.11"
 tonic-build = "0.8"
+anyhow = "1"
```

Comment:
anyhow and ethers are now twice included as deps

Reply:
This is for the `build.rs` only. Build.rs could only be imported in `[build-dependencies]`

Reply:
The `anyhow` in `[dependencies]` doesn't help in `build.rs`

Reply:
Yeah, my bad, looked at it too fast.

Reply:
Then we can alphabetically sort them so it's easier to find dep later.